### ISSUE TITLE
Updation Required in Installation wiki

### ISSUE THREAD

**Issue by Gh0sT-810** [facebook/mysql-5.6#1149]
Hi, I want to request for the updation on installation guide found at https://github.com/facebook/mysql-5.6/wiki/Build-Steps . As I am a beginner and I found it very hard to follow the guide for installing MySQL from https://github.com/facebook/mysql-5.6/tree/fb-mysql-8.0.17 branch.

**Comment by mdcallag** [comment]
I don't doubt you had problems as most or all of that is for MySQL 5.6. Can you update this to explain some of the problems? I am likely to build using 8.0.17 soon and want to learn about what I might encounter.

**Comment by mdcallag** [comment]
It is hard. It took me several hours and I have some experience building MyRocks. My notes on building are here:
http://smalldatum.blogspot.com/2020/10/mysql-8-is-greater-with-myrocks.html

But I think you are much better off using MyRocks from MariaDB or Percona Server rather than the FB branch, because MariaDB and Percona have large communities.

**Comment by Gh0sT-810** [comment]
> 
> 
> It is hard. It took me several hours and I have some experience building MyRocks. My notes on building are here:
> http://smalldatum.blogspot.com/2020/10/mysql-8-is-greater-with-myrocks.html
> 
> But I think you are much better off using MyRocks from MariaDB or Percona Server rather than the FB branch, because MariaDB and Percona have large communities.

Hi sorry for the late reply. Actually I have used oracle's MySQL till now and this was my first time using fb's MySQL version. I am a beginner in Developing apps and back-end services so most of the installation steps related to 8.0.17 on the internet are confusing. Thanks for sharing the link though I will try the steps written by you in installing.
